pkg/plugins/runtime/k8s/controllers/pod_controller.go

```diff
@@ -317,14 +318,27 @@ func (r *PodReconciler) createOrUpdateDataplane(
 	services []*kube_core.Service,
 	others []*mesh_k8s.Dataplane,
 ) error {
+	meshName := util_k8s.MeshOfByLabelOrAnnotation(r.Log, pod, ns)
+	k8sMesh := v1alpha1.Mesh{}
+	if err := r.Client.Get(ctx, kube_types.NamespacedName{Name: meshName}, &k8sMesh); err != nil {
+		return err
+	}
+	mesh := core_mesh.NewMeshResource()
+	if k8sMesh.Spec != nil {
+		r.Log.Info("GOT MESH", "mesh", string(k8sMesh.Spec.Raw))
```

Comment:
this wording needs to be improved here: it's not meaningful enough.

Reply:
ah, this should not be here, I forgot to remove it after debugging